site/posts/intake-esm-2020-3-16.ipynb

```diff
@@ -0,0 +1,483 @@
+    "    \"id\":\"aws-cesm1-le\",\n",
+    "    \"description\":\"This is an ESM collection for CESM1 Large Ensemble Zarr dataset publicly available on Amazon S3 (us-west-2 region)\",\n",
+    "    \"catalog_file\": \"sample-catalog.csv\",\n",
+    "    \"attributes\":[\n",
+    "        { \"column_name\":\"component\", \"vocabulary\":\"\"},\n",
+    "        { \"column_name\":\"frequency\", \"vocabulary\":\"\"},\n",
+    "        { \"column_name\":\"experiment\", \"vocabulary\":\"\"},\n",
+    "        { \"column_name\":\"variable\", \"vocabulary\": \"\"}\n",
+    "        ],\n",
+    "        \n",
+    "    \"assets\":{ \"column_name\":\"path\", \"format\":\"zarr\"},\n",
+    "    ...\n",
+    " }\n",
+    "\n",
+    "```\n",
+    "\n",
+    "`sample-catalog.csv`:\n",
+    "```\n",
+    "component,frequency,experiment,variable,path\n",
+    "atm,daily,20C,FLNS,s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FLNS.zarr\n",
+    "atm,daily,20C,FLNSC,s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FLNSC.zarr\n",
+    "```\n",
+    "    \n",
+    "    \n",
+    "For the example above, we have both `sample-collection.json` and `sample-catalog.csv` files. We can easily put the content of the `sample-catalog.csv` in the `sample-collection.json` file as follows:\n",
+    "    \n",
+    "\n",
+    "\n",
+    "`sample-collection.json`:\n",
+    "```json\n",
+    "{\n",
+    "    \"esmcat_version\":\"0.1.0\",\n",
+    "    \"id\":\"aws-cesm1-le\",\n",
+    "    \"description\":\"This is an ESM collection for CESM1 Large Ensemble Zarr dataset publicly available on Amazon S3 (us-west-2 region)\",\n",
+    "    \"catalog_dict\":[\n",
+    "        {\n",
+    "            \"component\":\"atm\",\n",
+    "            \"frequency\":\"daily\",\n",
+    "            \"experiment\":\"20C\",\n",
+    "            \"variable\":\"FLNS\",\n",
+    "            \"path\":\"s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FLNS.zarr\"\n",
+    "        },\n",
+    "        {\n",
+    "            \"component\":\"atm\",\n",
+    "            \"frequency\":\"daily\",\n",
+    "            \"experiment\":\"20C\",\n",
+    "            \"variable\":\"FLNSC\",\n",
+    "            \"path\":\"s3://ncar-cesm-lens/atm/daily/cesmLE-20C-FLNSC.zarr\"\n",
+    "        },\n",
+    "       ...\n",
+    "    ],\n",
+    "  \"attributes\":[\n",
+    "        { \"column_name\":\"component\", \"vocabulary\":\"\"},\n",
+    "        { \"column_name\":\"frequency\", \"vocabulary\":\"\"},\n",
+    "        { \"column_name\":\"experiment\", \"vocabulary\":\"\"},\n",
+    "        { \"column_name\":\"variable\", \"vocabulary\": \"\"}\n",
+    "        ],\n",
+    "        \n",
+    "    \"assets\":{ \"column_name\":\"path\", \"format\":\"zarr\"},\n",
+    "    ...\n",
+    "    \n",
+    "}\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "Now, we have to keep track of a single file (`sample-collection.json`) which `intake-esm` can parse:\n",
```

Comment:
Again, I would explain what the reader just saw:

```suggestion
    "And you can now see that the `catalog_file` entry has been replaced with the appropriate `catalog_dict` entry. 
 Now, we have to keep track of a single file (`sample-collection.json`) which `intake-esm` can parse:\n",
```

Again, just to highlight to the reader what they should be looking at.